CHANGELOG.md

```diff
@@ -14,7 +14,9 @@ _None_
 
 ### Bug Fixes
 
-_None_
+* .swift-version: fix .swift-version with Swift 5.2 to build in 6.2.0 or later.  
```

Comment:
That doesn't explain much:
```suggestion
* Update the Swift version in `.swift-version` so that the right version is used when building manually (using `swiftenv`).  
```